server/backend/quiz/tests/test_models.py

```
@@ -0,0 +1,49 @@
+import pytest
+from rest_framework.test import APIClient
```

Ruff (0.8.2)

`rest_framework.test.APIClient` imported but unused

Remove unused import: `rest_framework.test.APIClient`

(F401)